lib/training/training_base.rb

```diff
@@ -33,17 +33,10 @@ def self.load(slug_list: nil, content_class: self)
   def self.load_all
     TrainingLibrary.flush
     TrainingModule.flush
-    if Features.wiki_trainings?
-      TrainingModule.load
-      TrainingModule.all.each { |tm| TrainingSlide.load(slug_list: tm.slide_slugs) }
-      TrainingModule.flush
-      TrainingModule.load
-      TrainingLibrary.load
-    else
-      TrainingLibrary.load
-      TrainingModule.load
-      TrainingSlide.load
-    end
+    TrainingSlide.delete_all
```

Comment:
Did you find this to be necessary? I'd really like to avoid any situation where slides need to be deleted; the idea is that we overwrite the existing records instead, whenever we need to update them. The same should be true for the modules.

Reply:
I think we should only need to flush the TrainingLibrary after this. For module and slide, we can now just do `.load` and it should overwrite the old records.

But for wiki trainings, it might still be relevant to do `TrainingModule.all.each { |tm| TrainingSlide.load(slug_list: tm.slide_slugs) }` instead of Slide.load.